src/extensibility-and-integration/mobile-plugins/health-fitness/intro.md

```diff
@@ -113,6 +113,24 @@ To write health and fitness data you can use the **WriteData** action. Set the p
 
 To check that writing the health or fitness data on the device is working, verify the value of **WriteData.Success** is **True**.
 
+### Create logic to define a background job
+
+To define a background job you can use the SetBackgroundJob action. Set the parameters for the type of health or fitness variable you want to keep evaluating, define the notification trigger condition and its frequency, and define the notification content. 
+
+See below the parametrization of a background job for two different use cases:
```

Comment:
Change sentence:

Parametrization for two different use cases of a background job is shown below: